bin/spice/pkg/spec/spicepod_test.go

```diff
@@ -127,8 +127,8 @@ nested_unknown:
 	}
 
 	// Verify known fields
-	if spicePod.Version != "v1beta1" {
-		t.Errorf("Expected version v1beta1, got %s", spicePod.Version)
+	if spicePod.Version != "v1" {
```

Comment:
I think we should keep it backwards compatible and allow v1beta1

Reply:
it does, this is for the test where we write `v1`